pom.xml

```diff
@@ -40,6 +40,16 @@
             <version>5.9.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>2.0.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-nop</artifactId>
+            <version>2.0.5</version>
```

Comment:
please add 
<scope>test</scope>
for the implementation of slf4j.

please consider using https://mvnrepository.com/artifact/org.slf4j/slf4j-simple instead of slf4j-nop for the tests so that they produce output if that is desired. other possibilities are to use full fledged implementations and configure those to reduce amount of logged events.

Reply:
Added tests and changed to slf4j-simple from slf4j-nop